_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -0,0 +1,120 @@
+
+      <p>If you already have some experience with Apache Kafka or with Strimzi and Kubernetes, you can try to help others by answering questions or providing guidance and suggestions. 
+        You can also help by reviewing the documentation and either suggesting or contributing improvements. 
+        You can also help by making tutorials - for example as part of the documentation, a blog post or as a video.</p>
+
+      <p>Our documentation is currently in English. 
+        But not everyone speaks English perfectly. 
+        So you can also help by translating our documentation or blog posts into other languages.</p>
+
+      <h3>Provide Feedback and Improve Quality</h3>
+
+      <p>Feedback is important for. 
+        Reading about how great Strimzi is always feels great. 
+        But also negative feedback is important and helps us make Strimzi better. 
+        You can share your feedback for example on Slack or mailing lists.</p>
+
+      <p>But feedback can have many different forms. 
+        And one of the ways to provide feedback is for example also <a href="https://github.com/strimzi/strimzi-kafka-operator/issues/new/choose">reporting bugs, suggesting improvements or opening enhancements requests</a> for features you would like to see. 
+        You can also write <a href="https://github.com/strimzi/proposals">proposals</a> describing new features and how they should work and be implemented. 
+        You can do it even if you cannot or do not plan to implement them - someone else might.</p>
+
+      <p>You can also help us to improve the quality of our releases and help us test Strimzi. 
+        As most modern projects, we rely heavily on <a href="https://strimzi.io/blog/2020/09/21/introduction-to-system-tests/">automated tests</a>. 
+        You can always help us improve the tests - you can add new tests to improve the test coverage, improve their execution time and so on. 
+        ut you can also help by manually testing Strimzi in different configurations, environments and in different use-cases and report the results and any issues you found.</p>
+
+      <p>Every Strimzi release starts with a release candidate. 
+        Testing the RCs in your environments with your configurations and making sure everything works before the final release is done is also very helpful. 
+        Fixing any bugs before the release is always better and easier than doing patch releases.</p>
+
+      <h3>Write code</h3>
+
+      <p>Since Strimzi is a software project, you can of course also contribute by writing code. 
+        One of the things which makes Strimzi different from most other cloud native projects is that most of its parts are written in Java. 
+        So if you want to contribute to an exciting project and work with modern technologies such as Kubernetes but do not want to learn Golang, then Strimzi is the right project for you. 
```

Comment:
It's not true anymore now that we have canary in Golang but I got your main point around mostly the operator.

Reply:
It mentions the Canary and Golang few lines below. I think Java is one of the things which make us stand apart. So I think it is worth it to deliver the point in a clear way in this part of the page.